rotkehlchen/tests/api/test_current_assets_price.py

```diff
@@ -318,8 +318,8 @@ def test_manual_current_prices_loop(inquirer):
 
 @pytest.mark.vcr()
 @pytest.mark.parametrize('ignore_mocked_prices_for', ['ETH'])
-def test_inquirer_oracles_affect_manual_price(inquirer):
-    """Checks that change of oracles order affects manual current price usage.
+def test_inquirer_oracles_does_not_affect_manual_price(inquirer):
```

Comment:
Avove this it is failing in `rotkehlchen/tests/api/test_current_assets_price.py::test_manual_current_prices_loop` with 

```
AssertionError: assert FVal(0) == FVal(1570.92) + where FVal(1570.92) = FVal('1570.92')
```